src/main.cpp

```diff
@@ -22,7 +22,7 @@ void setup()
 {
     Serial.begin(baud_rate);
     Serial.println("setup");
-
+    //auto a = obc::init_lora();
```

Comment:
error trzeba obsluzyc a nie przypisac do zmiennej :P
podczas developmentu moezsz zrobic po prostu
```obc::init_lora().unwrap()```
co zrobi crasha, jest to tez jakas forma obsluzenia :D

Reply:
Docelowo funkcję `init_lora()` umieściłbym w pliku `devices.cpp` tam gdzie wykonane są pozostałe inity